README.md

```diff
@@ -1,4 +1,28 @@
-# snakeGame
-/* 🐍 🐍 🐍 🐍 🐍 */
 
-cool snake game with moveable food
+Welcome to snakeGame!
+===================
+A cool snake game with running food
+
+![alt text](https://github.com/srbcheema1/snakeGame/raw/master/src/images/picture1.png)
```

Comment:
image is not shown

Reply:
It will be after merge. Take a look: https://github.com/boaglio/snakeGame/blob/master/src/images/picture1.png